concept/Technical Concept.md

```diff
@@ -0,0 +1,18 @@
+Technical Concept
+=================
+
+These decisions have been made by the team:
+
+- Create a 2D game, because nobody of the team is familiar enough with 3D assets.
+
+- Use C#, because we are all very familiar with it.
+
+- Use Unity, as it popular, has a large ecosystem, and supports C# scripting as first class citizen.
+
+- Use a fixed aspect ratio for the game, because UI design and layouting would explode in effort otherwise.
+
+- Use 16:9 aspect ratio, because that's the most common.
+
+- Use mouse and keyboard for input devices (Reason?)
+
+- Make sure the game design could be ported to gamepad's in the future (Reason?)
```

Comment:
It's nice to keep doors open, I get that. But should it really be a constrain on our gameplay design? Do we expect to have enough time to add gamepad support as part of the project? Let's not focus too much on what's after the game jam. The jam is there, to force participants to focus on a particular idea. Not to make an idea accessible to every platform imaginable.

Reply:
I think a good game design either for pc or game pad should be easy and clear so I wouldn't call this it should be open for gamepad it just should be easy to unterstand, most other games that have a easy UI are good for pc and gamepad

Reply:
Do we want to consider touchscreen only for supporting mobile devices? Though I think introducing them would violate the 16:9 aspect ratio requirement since phones do whatever they want with resolutions.

Reply:
the 16:9 ratio can be kept but I would consider that they first shouldn't be implemented as that would take much time to test